### DISCUSSION TITLE
Change Item Spacing (Answered)

### DISCUSSION THREAD

**Discussion by BobBurgess** [totaldebug/atomic-calendar-revive#1483] [Showroom]
Hello,

I thought I'd post this as a separate entry in response to this question: https://github.com/totaldebug/atomic-calendar-revive/discussions/1472

That person asked to change the font size, which I had worked out how to do, but it results in a cramped look.

Is there a way to manually set the spacing of events to make them more separated for larger font sizes?

Regards,

**Comment by jackm96** [comment]
add the below

```
 ha-card {
    line-height: 150% !important
    }
```

**Reply by BobBurgess** [reply]
Thankyou. This works perfectly.

**Comment by BobBurgess** [comment]
Closing this discussion as the solution proposed by jackm96 works.
